contrib/build-push-ceph-container-imgs.sh

```diff
@@ -95,17 +95,12 @@ function install_docker {
 }
 
 function install_podman {
-  # https://github.com/containers/libpod/issues/5306
-  # https://podman.io/getting-started/installation.html
-  if ${CI_CONTAINER}; then
-    sudo dnf -y module disable container-tools
-    sudo dnf -y install 'dnf-command(copr)'
-    sudo dnf -y copr enable rhcontainerbot/container-selinux
-    sudo curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/CentOS_8/devel:kubic:libcontainers:stable.repo
-    # https://tracker.ceph.com/issues/44242
-    # We used to provide fuse-overlayfs-0.7.6-2.0 in lab-extras but a newer version is available in the kubic repo so we'll install/update from there
-    sudo dnf install -y fuse-overlayfs
-  fi
+  # we may need to undo prior packaging hacks
+  sudo dnf remove -y $(sudo dnf list | grep devel_kubic_libcontainers_stable \
+    | awk '{print $1;}')
+  sudo dnf config-manager --disable devel_kubic_libcontainers_stable
+  sudo dnf module enable -y container-tools:rhel8
+  # now install it
```

Comment:
@dmick  probably off-topic, could you add following line to ensure that libseccomp is updated? 

```sh
dnf upgrade libseccomp
```

because we have following errors like

```
/usr/bin/podman: symbol lookup error: /usr/bin/podman: undefined symbol: seccomp_api_get
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
```
when building images for aarch64. see https://jenkins.ceph.com/job/ceph-dev-build/ARCH=arm64,AVAILABLE_ARCH=arm64,AVAILABLE_DIST=centos8,DIST=centos8,MACHINE_SIZE=gigantic/46874//consoleFull

looks like it's https://github.com/containerd/containerd/issues/4619 .

Reply:
This doesn't affect podman 2.0.5, which is what this will revert to